### PR TITLE
Localization fix for license

### PR DESCRIPTION
**Pull request by garrett-eclipse**
In translations it's inconsistent with use of licence & license.
I tried to standardize by using en_US in strings and comments but didn't touch any variable/function references to avoid unforseen issues.

Fixes #

#### Changes proposed in this Pull Request:

* Standardize license verbiage for en_US

### CODE REVIEW

File: includes/helper/class-wp-job-manager-helper.php [hunk 1]
```diff
@@ -468,7 +468,7 @@ private function activate_licence( $product_slug, $licence_key, $email ) {
 	private function deactivate_licence( $product_slug ) {
 		$licence = $this->get_plugin_licence( $product_slug );
 		if ( empty( $licence['licence_key'] ) || empty( $licence['email'] ) ) {
-			$this->add_error( $product_slug, __( 'licence is not active.', 'wp-job-manager' ) );
+			$this->add_error( $product_slug, __( 'license is not active.', 'wp-job-manager' ) );
```
Comment (jom) [inline includes/helper/class-wp-job-manager-helper.php]: Good catch. Comments and option names use licence (WPJM's original dev was a Brit ;)) but all strings should use US spelling (license).

**Comment by garrett-eclipse** [comment]
Thanks @jom I've reverted the comments so this should just be the translatable strings. Cheers

**Comment by jom** [comment]
Thanks for the PR, @garrett-eclipse! Should be in the 1.29.1 release.

**Comment by garrett-eclipse** [comment]
Thanks @jom

**Pull request merged**